requirements/runtime-requirements.txt

```diff
@@ -25,4 +25,6 @@ zipstream==1.1.4
 h5py==2.7.1
 numpy==1.14.5
 celery==4.1.1
-django-webpack-loader==0.6.0
\ No newline at end of file
+pillow
+python-resize-image
+django-webpack-loader==0.6.0
```

Comment:
can you run `pip freeze` and put in the specific versions of the two new deps so that the build is reproducible

Reply:
Fixed.

Reply:
Fixed.